docs/3_development/2_tech-stack/Typescript/monorepo.md

```diff
@@ -27,6 +27,13 @@ If you are running node@16+, you just need to run `corepack enable`.
 If you are running older version, run `npm i -g corepack`.
 Make sure you put this in README file for future contributors.
 
+:::warning
+
+Date: 22 November 2023
+There is an issue with `actions/setup-node@v3` it is used with `cache: 'yarn'`
```

Comment:
```suggestion
There is an issue with `actions/setup-node@v3` if is used with `cache: 'yarn'`
```
can you link issue?

Reply:
there is no exact issue, but here is some info
https://github.com/actions/setup-node/issues/480#issuecomment-1820622085

Reply:
I see, maybe better to add instructions to install cache in separate step (after corepack enable)